### PR TITLE
Several fixes

### PR DESCRIPTION
**Pull request by ldm0**
cc @peterli-r3 
The sample and template all updated a bit.... Here are some of the problems I found. Please review your tutorials....

### CODE REVIEW

**Comment by ivanterziev-r3** [comment]
NB @taylorharvey-r3 @AnnaK-R3 This needs propagated to 4.8 as well once merged.

**Review by AnnaK-R3 — APPROVED** [review]

**Review by taylorharvey-r3 — APPROVED** [review]
LGTM but I'd like @peterli-r3 to validate as well.

File: content/en/docs/corda-os/4.7/tutorial-cordapp.md [hunk 1]
```diff
@@ -546,7 +546,7 @@ This will print out the following progress steps:
 We can also issue RPC operations to the node via the interactive shell. Type `run` to see the full list of available
 operations.
 
-You can see the newly-created IOU by running `run vaultQuery contractStateType: com.example.state.IOUState`.
+You can see the newly-created IOU by running `run vaultQuery contractStateType: net.corda.samples.example.states.IOUState`.
 
 As before, the interactive shell of PartyC will not display any IOUs.
```
Comment (peterli-r3) [inline content/en/docs/corda-os/4.7/tutorial-cordapp.md:551]: This page is currently worked by Erin on https://github.com/corda/corda-docs/pull/438 as well. 
I would suggest merging this first. and then Erin's updates.

File: content/en/docs/corda-os/4.7/tutorial-cordapp.md [hunk 2]
```diff
@@ -430,10 +430,10 @@ These ports are defined in `clients/build.gradle`.
...
 
 
-* `/api/example/me`
-* `/api/example/peers`
-* `/api/example/ious`
-* `/api/example/create-iou` with parameters `iouValue` and `partyName` which is CN name of a node
+* `/me`
+* `/peers`
+* `/ious`
+* `/create-iou` with parameters `iouValue` and `partyName` which is CN name of a node
```
Comment (peterli-r3) [inline content/en/docs/corda-os/4.7/tutorial-cordapp.md:437]: This is correct.

File: content/en/docs/corda-os/4.7/tutorial-cordapp.md [hunk 3]
```diff
@@ -448,13 +448,13 @@ anti-XSS, anti-XSRF or other security techniques. Do not use this code in produc
 
 #### Creating an IOU via the endpoint
 
-An IOU can be created by sending a PUT request to the `/api/example/create-iou` endpoint directly, or by using the
+An IOU can be created by sending a PUT request to the `/create-iou` endpoint directly, or by using the
 the web form served from the home directory.
```
Comment (peterli-r3) [inline content/en/docs/corda-os/4.7/tutorial-cordapp.md:453]: correct

File: content/en/docs/corda-os/4.7/tutorial-cordapp.md [hunk 4]
```diff
@@ -448,13 +448,13 @@ anti-XSS, anti-XSRF or other security techniques. Do not use this code in produc
...
 
 To create an IOU between PartyA and PartyB, run the following command from the command line:
 
 ```bash
-curl -i -X POST 'http://localhost:50005/api/example/create-iou?iouValue=12&partyName=O=PartyB,L=New%20York,C=US' -H 'Content-Type: application/x-www-form-urlencoded'
+curl -i -X POST 'http://localhost:50005/create-iou?iouValue=12&partyName=O=PartyB,L=New%20York,C=US' -H 'Content-Type: application/x-www-form-urlencoded'
 ```
```
Comment (peterli-r3) [inline content/en/docs/corda-os/4.7/tutorial-cordapp.md:459]: correct

File: content/en/docs/corda-os/4.7/tutorial-cordapp.md [hunk 5]
```diff
@@ -482,8 +482,8 @@ Assuming all went well, you can view the newly-created IOU by accessing the vaul
 *Via the HTTP API:*
 
 
-* PartyA’s vault: Navigate to [http://localhost:50005/api/example/ious](http://localhost:50005/api/example/ious)
-* PartyB’s vault: Navigate to [http://localhost:50006/api/example/ious](http://localhost:50006/api/example/ious)
+* PartyA’s vault: Navigate to [http://localhost:50005/ious](http://localhost:50005/ious)
+* PartyB’s vault: Navigate to [http://localhost:50006/ious](http://localhost:50006/ious)
```
Comment (peterli-r3) [inline content/en/docs/corda-os/4.7/tutorial-cordapp.md:487]: correct

File: content/en/docs/corda-os/4.7/tutorial-cordapp.md [hunk 6]
```diff
@@ -546,7 +546,7 @@ This will print out the following progress steps:
 We can also issue RPC operations to the node via the interactive shell. Type `run` to see the full list of available
 operations.
 
-You can see the newly-created IOU by running `run vaultQuery contractStateType: com.example.state.IOUState`.
+You can see the newly-created IOU by running `run vaultQuery contractStateType: net.corda.samples.example.states.IOUState`.
```
Comment (peterli-r3) [inline content/en/docs/corda-os/4.7/tutorial-cordapp.md:550]: correct

**Pull request merged**